src/nav/routingSelectors.js

```diff
@@ -0,0 +1,25 @@
+export const getInitialRoutes = (accounts: any[]): string => {
+  const activeAccount = accounts.get(0);
+
+  if (activeAccount && activeAccount.get('apiKey')) {
+    return ['main'];
+  }
+
+  const routeList = [];
+
+  if (accounts.size > 1) routeList.push('accountlist');
+
+  routeList.push('realm');
+
+  if (activeAccount) {
+    if (activeAccount.get('email')) {
+      routeList.push('password');
+    } else if (activeAccount.get('todo')) {
```

Comment:
Let's get rid of the 'todos' and add this logic back when ready (just so that anyone looking at this commit in the future isn't confused).